pages/common/atool.md

```diff
@@ -0,0 +1,37 @@
+# atool
+
+> A script for managing file archives of various types.
+> `atool` uses external archiver programs but provides a consistent command-line interface for listing, extracting, creating, and managing archives.
+> More information: <https://savannah.nongnu.org/projects/atool/>.
+
+- List files in an archive:
+
+`atool {{[-l|--list]}} {{path/to/archive.zip}}`
+
+- Extract an archive (safely creates a subdirectory if needed):
+
+`atool {{[-x|--extract]}} {{archive.tar.gz}}`
+
+- Extract an archive to a specific directory:
+
+`atool -X {{path/to/output_directory}} {{archive.rar}}`
```

Comment:
```suggestion
`atool {{[-X|--extract-to]}} {{path/to/output_directory}} {{archive.rar}}`
```

Reply:
AFAIK Semantics of --extract-to work a bit differently than -X and -X doesn't need equals sign = and --extract-to needs it. Because of this they don't seem to be interchangeable like this but for me it doesn't really matter 🤷

Reply:
Documentation oftentimes uses =, but in reality allows both = and space. We have it in our styleguide to prefer using spaces unless the = is a must

Reply:
I went ahead and tested it myself. It's a valid command.